templates/hosting/init.js

```diff
@@ -1,5 +1,31 @@
 if (typeof firebase === 'undefined') throw new Error('hosting/init-error: Firebase SDK not detected. You must include it before /__/firebase/init.js');
 /*--CONFIG--*/
+/*--EMULATORS--*/
```

Comment:
Should this go with the `if (firebaseEmulators)` line?

Reply:
Yep